common/src/types.ts

```diff
@@ -1,4 +1,11 @@
-import { EncryptedShortString, PublicKey, RandomString, Signature } from "./crypto.ts";
+export { err, ok, Result } from "npm:true-myth@6.2.0/result";
```

Comment:
make sure when you merge, the new functions are used (aka no dead code)

Reply:
👍